cardano-api/src/Cardano/Api/Modes.hs

```diff
@@ -50,11 +53,8 @@ import qualified Ouroboros.Consensus.Cardano.Block as Consensus
 import qualified Ouroboros.Consensus.Cardano.ByronHFC as Consensus (ByronBlockHFC)
 import           Ouroboros.Consensus.HardFork.Combinator as Consensus (EraIndex (..), eraIndexSucc,
                    eraIndexZero)
-import           Ouroboros.Consensus.Shelley.Eras
-                   (StandardShelley,
-                    StandardAllegra,
-                    StandardMary,
-                    StandardAlonzo)
+import           Ouroboros.Consensus.Shelley.Eras (StandardAllegra, StandardAlonzo, StandardMary,
+                   StandardShelley)
```

Comment:
When changing other people's code, try to avoid the temptation to include things that your particular code tidying tool does, like random whitespace changes, or in this case changing the eras from chronological order to alphabetical order.

Reply:
Right, sorry about that. I'll revert it.